### PR TITLE
Blog about Open Spaces added

### PR DESCRIPTION
**Pull request by realslimshanky**

### CODE REVIEW

**Comment by akshayaurora** [comment]
👍

File: content/blog/understading-open-spaces.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Title: Understanding Open Spaces
+Date: 2017-08-29 11:00
+Category: 2017
+Slug: understanding-open-spaces
+Summary: understanding-open-spaces
+
+Pycon India is back! It’s time to meet some amazing Pythonistas, share their experiences and learn a ton from some really inspiring like-minded open source enthusiasts. We are happy to inform you that we have decided to continue with our tradition of open space this year as well. Open spaces are not only a great way to create a vibrant community but also a really useful mode of sharing knowledge and learning from the same.
```
Comment (sayanchowdhury) [inline content/blog/understading-open-spaces.md]: Can we change Pycon to PyCon here?

Comment (realslimshanky) [reply content/blog/understading-open-spaces.md]: Sure, I'll add this fix to the next PR.

File: content/blog/understading-open-spaces.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+###What is an Open Space?
+
+Literally, it's a SPACE OPEN to anyone who wishes to talk about Python. Be it experience, learnings, libraries, tools, frameworks, tests, correct code practices, an idea, a "What-not-to-do" talk or anything (in this case about Python).
```
[the remark this answers is not in the document]
Comment (nupuragrahari) [reply content/blog/understading-open-spaces.md]: Seemed to couple better with "literally". Nothing else really!

**Pull request merged**